### PR TITLE
UX: Do not delete narrative bot PM when skipping user tips

### PR DESCRIPTION
**Pull request by pmusaraj**
Should fix https://meta.discourse.org/t/skip-tips-option-in-user-tip-should-not-delete-welcome-message/303420

### CODE REVIEW

File: plugins/discourse-narrative-bot/spec/system/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      SiteSetting.discourse_narrative_bot_enabled = true
+      # shortcut to generate welcome post since we're not going through user creation or first login
+      user.enqueue_bot_welcome_post
+    end
+
+    it "does not delete the narrative bot PM when skipping all tips" do
+      sign_in user
+      visit "/"
+
+      find(".fk-d-tooltip__content .user-tip__buttons .btn", text: "Skip tips").click
```
Comment (jjaffeux) [inline plugins/discourse-narrative-bot/spec/system/user_spec.rb:22]: I should write a page object for this :)

**Review by jjaffeux — APPROVED** [review]

File: plugins/discourse-narrative-bot/spec/system/user_spec.rb (hunk 1 above)
Comment (jjaffeux) [reply plugins/discourse-narrative-bot/spec/system/user_spec.rb:22]: I did it: https://github.com/discourse/discourse/commit/2fdee7849cc6c79747f6cd2fa607bf6372ee1dd9

File: plugins/discourse-narrative-bot/spec/system/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    end
+
+    it "does not delete the narrative bot PM when skipping all tips" do
+      sign_in user
+      visit "/"
+
+      tooltip = PageObjects::Components::Tooltips.new("user-tip")
+      tooltip.find(".btn", text: "Skip tips").click
+
+      expect(page).to have_no_css(".fk-d-tooltip__content .user-tip__title")
```
Comment (jjaffeux) [inline plugins/discourse-narrative-bot/spec/system/user_spec.rb:25]:
```suggestion
      expect(tooltip).to be_not_present
```

**Review by jjaffeux — APPROVED** [review]

File: plugins/discourse-narrative-bot/spec/system/user_spec.rb (hunk 2 above)
Comment (pmusaraj) [reply plugins/discourse-narrative-bot/spec/system/user_spec.rb]: Duh! Thanks!

File: plugins/discourse-narrative-bot/spec/system/user_spec.rb (hunk 1 above)
Comment (pmusaraj) [reply plugins/discourse-narrative-bot/spec/system/user_spec.rb]: Thanks!

**Pull request merged**